src/test/java/uk/gov/ida/eventemitter/EventEmitterAmazonEventSenderTest.java

```diff
@@ -0,0 +1,124 @@
+
+import java.net.URI;
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static uk.gov.ida.eventemitter.EventEmitterTestHelper.AUDIT_EVENTS_API_RESOURCE;
+import static uk.gov.ida.eventemitter.EventEmitterTestHelper.createTestResponseHeadersMap;
+import static uk.gov.ida.eventemitter.utils.TestEventBuilder.aTestEventMessage;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EventEmitterAmazonEventSenderTest {
+
+    private static final String DUMMY_AWS_ACCESS_KEY = "ABCD1234EFGH5678IJKL";
+    private static final String DUMMY_AWS_SECRET_ACCESS_KEY = "abcd1234DEFH5678ijkl4321mnop8765qrst1234";
+
+    @ClassRule
+    public static HttpStubRule apiGatewayStub = new HttpStubRule();
+
+    @Before
+    public void setup() {
+        apiGatewayStub.reset();
+    }
+
+    @After
+    public void tearDown() {
+        apiGatewayStub.reset();
+    }
+
+    @Test
+    public void shouldReturnHTTP403WhenCredentialsMissing() throws Exception {
+
+        final RegisteredResponse expectedResponse = new RegisteredResponse(
+                403,
+                "application/json",
+                "hello",
+                createTestResponseHeadersMap()
+        );
+
+        final EventSender amazonEventSender =
+                new AmazonEventSender(URI.create(apiGatewayStub.baseUri().build().toString() + AUDIT_EVENTS_API_RESOURCE),
+                        new BasicAWSCredentials("", ""),
+                        Regions.EU_WEST_2);
+
+        final ExpectedRequest expectedRequest = new ExpectedRequest(AUDIT_EVENTS_API_RESOURCE, "POST", null, null);
+
+        try {
+            apiGatewayStub.register(expectedRequest, expectedResponse);
+            Response<Void> response = amazonEventSender.sendAuthenticated(aTestEventMessage().build(), "encryptedEvent");
+            Fail.fail("Should return Forbidden (403)");
+        } catch (AwsResponseException e) {
+            assertThat(e.getResponse().getStatusCode()).isEqualTo(403);
+        }
+    }
+
+    @Test
+    public void shouldReturnHTTP200WhenAuthenticationSucceeds() throws Exception {
+
+        final RegisteredResponse expectedResponse = new RegisteredResponse(
+                200,
+                "application/json",
+                "hello",
+                createTestResponseHeadersMap()
+        );
+        final EventSender amazonEventSender =
+                new AmazonEventSender(URI.create(apiGatewayStub.baseUri().build().toString() + AUDIT_EVENTS_API_RESOURCE),
+                        new BasicAWSCredentials(DUMMY_AWS_ACCESS_KEY, DUMMY_AWS_SECRET_ACCESS_KEY),
+                        Regions.EU_WEST_2);
+
+        /*
+            Expected headers not provided as HttpStub requires both header names and content to match, and content is dynamic.
+            The expected headers are tested separated using assertions in the try block below.
+         */
+        final ExpectedRequest expectedRequest =
```

Comment:
Duplicate code.

Reply:
`final ExpectedRequest expectedRequest = new ExpectedRequest(AUDIT_EVENTS_API_RESOURCE, "POST", null, null);` appear in both methods `shouldReturnHTTP403WhenCredentialsMissing()` and `shouldReturnHTTP200WhenAuthenticationSucceeds`. We can make make this static final variable`expectedRequest` as part of EventEmitterAmazonEventSenderTest class.